src/components/CrucibleFilter.vue

```diff
@@ -65,17 +67,26 @@
 </template>
 
 <script setup lang="ts">
-import { ref, computed } from "vue";
+import { ref, computed, inject, onMounted } from "vue";
 import CollapseBtn from "./CollapseBtn.vue";
 import FilterButton from "./FilterButton.vue";
 //ToDo: inject the taxonomyTags from the Crucible Main platform
-import { taxonomyGroups } from "./DataAccessLayer";
+import { staticFilterSetTags } from "./DataAccessLayer";
+import { getFilterSetTags } from "./DataAccessLayer";
+const emit = defineEmits(["updateFilterTagArray"]);
+
+const filterSetApi =
+  (inject("$filterSetApi") as string) ||
+  "http://localhost:8080/api/resource/getFilterSet";
+
+const filterDataApi =
+  (inject("$filterDataApi") as string) ||
+  "http://localhost:8080/api/resource/getFilterData";
```

Comment:
Is this Api created or does it need to be created?

Reply:
yes, will delete this, thanks